scripts/repl.js

```diff
@@ -212,8 +212,9 @@
     }
   };
 
+  var capturingConsole;
```

Comment:
No, the whole point of this pull request is tho **disable** the instance that was given to the previous version of the code.  This works because there is a check to see if `this === capturingConsole`